### PR TITLE
docs: added api page for system audit and updated integrations pages with requirements

### PR DESCRIPTION
**Pull request by robbyandrews**

### CODE REVIEW

File: content/intro/integrations/messaging/slack/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  - slack
+resources:
+  - src: code-risk-alerts-in-slack-via-blubracket.png
+---
+
+### Slack Integration
+
+{{< figure src="code-risk-alerts-in-slack-via-blubracket.png" caption="BluBracket integration with Slack can alert teams about code risks in new commits in real time." >}}
+
+### Requirements
```
Comment (misterbisson) [inline content/intro/integrations/messaging/slack/index.md:21]: Should these pages be in http://docs.blubracket.com/how-to/integrations instead?

File: content/how-to/ticketing/jira/index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+### Requirements
+
+* [Atlassian API Token](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/)
+
+* [JIRA Base URL](https://confluence.atlassian.com/adminjiraserver/configuring-the-base-url-938847830.html)
+
+* [JIRA Project for events and alerts to be added to.](https://support.atlassian.com/jira-software-cloud/docs/create-a-new-project/)
+
+If you are interested in integrating JIRA into your BluBracket workflow please [contact support.](mailto:support@blubracket.com)
```
Comment (misterbisson) [inline content/how-to/ticketing/jira/index.md:29]: Email or link to http://docs.blubracket.com/contact-support/ ?

File: content/how-to/messaging/slack/index.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Integration with Slack requires the following:
+
+* [Slack API Key](https://slack.com/help/articles/215770388-Create-and-regenerate-API-tokens)
+
+* [Valid Slack Workplace URL](https://slack.com/help/articles/221769328-Locate-your-Slack-URL)
+
+* [A destination Slack channel for alerts to be posted.](https://slack.com/integrations)
+
+If you are interested in integrating Slack into your BluBracket workflow please [contact support.](mailto:support@blubracket.com)
```
Comment (misterbisson) [inline content/how-to/messaging/slack/index.md:31]: Email or link to http://docs.blubracket.com/contact-support/ ?

File: content/api/system-audit/index.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
+---
+title: "System Audit"
+description: "Auditing list of users who have accessed the Blubracket System"
```
Comment (kathytong) [inline content/api/system-audit/index.md:3]: nit: BluBracket

**Review by misterbisson — APPROVED** [review]

**Pull request merged**